fontc/src/main.rs

```diff
@@ -4,21 +4,27 @@ use clap::Parser;
 
 use fontbe::orchestration::AnyWorkId;
 use fontc::{create_timer, Args, Error, JobTimer};
+use log::{error, warn};
 
 fn main() {
     let args = Args::parse();
-    let verbose = args.verbose;
 
     // catch and print errors manually, to avoid just seeing the Debug impls
     if let Err(e) = run(args) {
+        let mut error_displayed = false;
+        let mut additional = "";
         if let Error::Backend(fontbe::error::Error::FeaCompileError(e)) = &e {
-            if verbose {
-                let _ = writeln!(std::io::stderr(), "{}", e.display_verbose());
-                std::process::exit(1);
+            if log::log_enabled!(log::Level::Warn) {
+                warn!("{}", e.display_verbose());
```

Comment:
the call of `e.display_verbose()` is required in order to see the details of the error that you are wanting to see (the full diagnostic message for each error encountered in the FEA)

Reply:
We're still making the call, it's just emitted via the logging system.